cgrok/grok.go

```diff
@@ -1,8 +1,9 @@
 package grok
 
 /*
-#cgo LDFLAGS: -lgrok
-#include <grok.h>
+#cgo CFLAGS: -I. -std=gnu99
+#cgo windows LDFLAGS: -L. -lportablexdr -lws2_32
+#include "grok.h"
```

Comment:
I'm not familiar with what go is doing here, but it looks like it's providing some flags for the cgo tool. Does this change the location where it looks for `grok.h` from the system include directories to a relative directory?